document/core/text/instructions.rst

```diff
@@ -926,14 +929,20 @@ Such a folded instruction can appear anywhere a regular instruction can.
      \text{(}~\text{block}~~\Tlabel~~\Tblocktype~~\Tinstr^\ast~\text{)}
        &\equiv\quad \text{block}~~\Tlabel~~\Tblocktype~~\Tinstr^\ast~~\text{end} \\ &
      \text{(}~\text{loop}~~\Tlabel~~\Tblocktype~~\Tinstr^\ast~\text{)}
-       &\equiv\quad \text{loop}~~\Tlabel~~\Tblocktype~~\Tinstr^\ast~~\text{end} \\ &
-     \text{(}~\text{if}~~\Tlabel~~\Tblocktype~~\Tfoldedinstr^\ast
-       &\hspace{-3ex} \text{(}~\text{then}~~\Tinstr_1^\ast~\text{)}~~\text{(}~\text{else}~~\Tinstr_2^\ast~\text{)}^?~~\text{)}
-       \quad\equiv \\ &\qquad
-         \Tfoldedinstr^\ast~~\text{if}~~\Tlabel~~\Tblocktype &\hspace{-1ex} \Tinstr_1^\ast~~\text{else}~~(\Tinstr_2^\ast)^?~\text{end} \\ &
+       &\equiv\quad \text{loop}~~\Tlabel~~\Tblocktype~~\Tinstr^\ast~~\text{end} \\
+   \end{array}\\
+   \begin{array}{lr}
+     \text{(}~\text{if}~~\Tlabel~~\Tblocktype~~\Tfoldedinstr^\ast~~
+       \text{(}~\text{then}~~\Tinstr_1^\ast~\text{)}~~\text{(}~\text{else}~~\Tinstr_2^\ast~\text{)}^?~~\text{)}
+       & \equiv \\
+       \qquad \Tfoldedinstr^\ast~~\text{if}~~\Tlabel~~\Tblocktype~~\Tinstr_1^\ast~~\text{else}~~(\Tinstr_2^\ast)^?~~\text{end} &\\
      \text{(}~\text{try}~~\Tlabel~~\Tblocktype~~\text{(}~\text{do}~~\Tinstr_1^\ast~~\text{)}
```

Comment:
```suggestion
```

As I suggested in https://github.com/WebAssembly/exception-handling/pull/180#discussion_r832889701, you should delete this line after applying that patch. The current equation is broken and doesn't build

Reply:
Gentle ping :) This line 939 needs to be deleted. The current code repeats `(try label blocktype (do instr1)` twice.
<img width="890" alt="image" src="https://user-images.githubusercontent.com/8726997/181218907-58682dcc-eb96-4dfb-a692-74c785b28cf2.png">

Reply:
As I said in another comment, `(delegate)` doesn't have instructions within it. Let's continue that discussion there; Will mark this thread as resolved.